.ci/variables.sh

```diff
@@ -1,2 +1,6 @@
+#!/bin/bash
+
 export PACKAGE='github.com/m3db/m3db'
+export VENDOR_PATH=$PACKAGE/vendor
+export LICENSE_BIN=$GOPATH/src/$VENDOR_PATH/github.com/uber/uber-licence/bin/licence
```

Comment:
Yeah either that, or we roll out our own license fixer (e.g., some crude text prepending thing). FWIW gwr also does this.